index.html

```diff
@@ -305,9 +305,12 @@ <h2>Sponsors</h2>
                             </a>
+                            <a href="https://www.kanoma.fr" target="_blank" rel="noopener noreferrer">
+                                <img src="img/sponsors/kanoma.png" alt="Kanoma" />
```

Comment:
je mettrais une largeur ici aussi, pour s'assurer qu'on a bien la taille voulue (la même que pour Wijin tant qu'à faire)
```suggestion
                                <img src="img/sponsors/kanoma.png" alt="Kanoma" width="320px" />
```

Reply:
parfait